### PR TITLE
[#12081] User-friendliness: Fix evaluee/recipient mobile text wrap

### PR DESCRIPTION
**Pull request by jasonqiu212**
Part of #12081 
Sub-issue: [Submit feedback page: Fix text wrap of "Evaluee/Recipient" on mobile for all questions](https://github.com/TEAMMATES/teammates/projects/16#card-88047608)

**Outline of Solution**

- Problem: Text was wrapped due to `col-2` bootstrap styling.
- Solution: Change to use `col` instead

### CODE REVIEW

**Comment by weiquu** [comment]
Looks good! Small nit: wondering if the evaluee/recipient should be aligned to the left to be consistent with the non-mobile version and serve as a visual indicator that it is a sort of header. Comparison (mobile vs non-mobile):

Mobile

Not mobile

but small issue bah (:

File: src/web/app/components/question-submission-form/question-submission-form.component.scss [hunk 1]
```diff
@@ -27,25 +27,13 @@
...
-  padding-right: 50px;
-}
-
-@media (max-width: 768px) {
-  .evaluee-col {
-    padding-left: 5px;
-    padding-right: 5px;
-  }
-}
-
```
Comment (samuelfangjw) [inline src/web/app/components/question-submission-form/question-submission-form.component.scss:41]: 👍

File: src/web/app/components/question-submission-form/question-submission-form.component.scss [hunk 2]
```diff
@@ -27,25 +27,13 @@
...
 .evaluee-select {
   padding-left: 15px;
   padding-right: 15px;
 }
 
 .visibility-card {
-  background-color: rgba(0, 0, 0, .03);
+  background-color: rgba(0, 0, 0, 0.03);
```
Comment (zhaojj2209) [inline src/web/app/components/question-submission-form/question-submission-form.component.scss:36]: Turns out adding this causing linting to fail :P Do revert this.

**Review by zhaojj2209 — COMMENTED** [review]
One last nit

File: src/web/app/components/question-submission-form/question-submission-form.component.scss (hunk 2 above)
Comment (jasonqiu212) [reply src/web/app/components/question-submission-form/question-submission-form.component.scss]: Ahh sorry, did not catch this! Will fix asap

**Review by zhaojj2209 — APPROVED** [review]
LGTM

**Pull request merged**